.storybook/components/Docs/Guidelines/CodeGuidelines.mdx

```diff
@@ -135,20 +135,6 @@ We use focus-visible for most focus states. This ensures that the focus state wi
 
 However, we currently support some browser versions that do not support the focus-visible CSS feature, so we also use a fallback block in conjunciton with focus-visible.
 
-```css
```

Comment:
Moving away from using CSS module mixins, so that the individual css files are **just** css (useful for future efforts)